lib/clever-ruby/models/section.rb

```diff
@@ -50,17 +50,38 @@ class Section
 
     attr_accessor :term
 
+    class EnumAttributeValidator
```

Comment:
It seems like we could pull out EnumAttributeValidator into a mixin instead of repeating it everywhere. But probably not worth it if it's all auto-generated